matcher/service/openreview_interface.py

```diff
@@ -233,36 +265,19 @@ def set_alternates(self, alternates_by_forum):
 
     def _get_quota_arrays(self):
         '''get `minimum` and `maximum` reviewer load arrays, accounting for custom loads'''
-        minimums = [int(self.config_note.content['min_papers']) for r in self.reviewers]
```

Comment:
why did you change this implementation?

Reply:
`r` was an unused variable and causes pylint to throw a warning on that. Replacing unused variables with underscores is in accordance with PEP8 conventions.